interactive/interactive/tools.py

```diff
@@ -46,8 +46,13 @@ def open_json(filename):
     with open(path) as f:
         dicti = json.loads(f.read())
     return dicti
-
-
+def direct_open_json(filePath):
+    with open(filePath) as f:
+        dicti = json.loads(f.read())
+    return dicti
+def direct_dump_json(filePath, content):
```

Comment:
`direct_open_json` and `direct_dump_json` are used sometimes in the code and sometimes not. I am in favor of these function that open and dump the json, but can we use them consistently when reading json throughout the code?

Reply:
yea so direct_open_json is used for multi-user stuff. the regular open_json is for single users because we wanted to keep the single-user experience working perfectly until multi-user is completely done -- so it'll be streamlined/consistent later on once old code is removed

Reply:
So all of the times that a json file is opened in the code without this function happens only for single users?

Reply:
99% of the time yes, some functions open a json file themselves if there's some special option they need or something

Reply:
Then make this function take `**kwargs` as arguments and pass those to `json.loads`?